Extending/Key-Vault/index.md

```diff
@@ -62,6 +69,119 @@ There are different ways to access the Azure Key Vault. It is important that the
 1. Search for the user. 
 1. Click review + assign
 
+## Directly reference a secret in Azure Web Apps
```

Comment:
As this feature is called "Key Vault references" I would recommend using that in the title, perhaps "Use Key Vault references for Azure App Services"

Reply:
Also perhaps add a link to https://learn.microsoft.com/en-us/azure/app-service/app-service-key-vault-references

Reply:
Thanks Jeavon, this is now updated.